src/physics/euler.h

```diff
@@ -76,6 +76,16 @@ namespace Physics {
 template <int dim, int nstate, typename real>
 class Euler : public PhysicsBase <dim, nstate, real>
 {
+protected:
+    // For overloading the virtual functions defined in PhysicsBase
+    /** Once you overload a function from Base class in Derived class,
+     *  all functions with the same name in the Base class get hidden in Derived class.  
+     *  
+     *  Solution: In order to make the hidden function visible in derived class, 
+     *  we need to add the following:
```

Comment:
Is this comment missing something? What do we add "in order to make the hidden function visible in derived class"?

Reply:
Its there, the two following lines of code

Reply:
Hmm ok, but it might not appear in the doxygen documentation.

Reply:
Oh, nvm. Got it.